apis/extension/scheduling.go

```diff
@@ -33,6 +32,24 @@ const (
 	AnnotationReservationAllocated = SchedulingDomainPrefix + "/reservation-allocated"
 )
 
+//Gang scheduling
+const (
+	//Gang Mode
+	GangModePrefix = "GangMode"
+	StrictMode     = GangModePrefix + "Strict"
+	NonStrictMode  = GangModePrefix + "NonStrict"
+
+	// Gang's Annotation
+	AnnotationGangPrefix   = "gang.scheduling.koordinator.sh"
```

Comment:
should we put this domain const under `apis/extension/constants.go`? WDYT @eahydra

Reply:
Hmm... That's fine too..